apps/api/src/domain/games/memory-match/services/memory-match.service.ts

```diff
@@ -173,6 +173,19 @@ export class MemoryMatchService {
         return id;
     }
 
+    async removeCard(
+        roundId: AggregateId,
+        sequenceNumber: number
+    ): Promise<ResultOrError<AggregateId>> {
+        const result = await this.memoryMatchRepository.removeCard(roundId, sequenceNumber);
+
+        if (result instanceof Error) {
```

Comment:
Ah this wrapping of the error is unfortunate. Let's see if there' something we can do about this once everything else is working.